momento/simple_cache_client.go

```diff
@@ -28,12 +60,12 @@ type SimpleCacheClientProps struct {
 	DefaultTTL         time.Duration
 }
 
-// NewSimpleCacheClient returns a new ScsClient with provided authToken, DefaultTTLSeconds, and opts arguments.
-func NewSimpleCacheClient(props *SimpleCacheClientProps) (*ScsClient, error) {
+// NewSimpleCacheClient returns a new defaultScsClient with provided authToken, DefaultTTLSeconds, and opts arguments.
```

Comment:
This conflicts with the return signature. It's a user facing method and should not promise to return an internal struct.

```suggestion
// NewSimpleCacheClient returns a new SimpleCacheClient with provided authToken, DefaultTTLSeconds, and opts arguments.
```